toolchain/add.go

```diff
@@ -32,7 +32,8 @@ func Add(dir, toolchainPath string, opt *AddOpt) error {
 		return err
 	}
 
-	targetDir := filepath.Join(srclib.PathEntries()[0], toolchainPath)
```

Comment:
Actually, it's being used by the `Dir` function:

```
func Dir(toolchainPath string) (string, error) {
	toolchainPath = filepath.Clean(toolchainPath)

	dir, err := lookupToolchain(toolchainPath)
	if os.IsNotExist(err) {
		return filepath.Join(srclib.PathEntries()[0], toolchainPath), nil
	}
	if err != nil {
		err = &os.PathError{Op: "toolchain.Dir", Path: toolchainPath, Err: err}
	}
	return dir, err
}
```

Should `srclib.PathEntries()[0]` in the definition above be replaced with `filepath.SplitList(srclib.Path)[0]` so we can remove `PathEntries`?